### ISSUE TITLE
Config.js sample configuration

### ISSUE THREAD

**Issue by jacobabiola** [poanetwork/ico-wizard#478]
If you are reporting a problem with ICO Wizard, please include the following information:

### Which network did you use? (Mainnet, Kovan, Rinkeby, etc.)

*Your answer*
sokol RPC on a Localhost project
### If you were able to create it, what is the URL of your crowdsale?

*Your answer*
Local host
### Do you have screenshots showing the problem?

*Your answer*
![screen shot 2018-01-11 at 11 18 08 am](https://user-images.githubusercontent.com/24619534/34820605-4ed3c4cc-f6c1-11e7-98e4-35355e7a259f.png)
![screen shot 2018-01-11 at 11 18 27 am](https://user-images.githubusercontent.com/24619534/34820606-4f04c09a-f6c1-11e7-8ae6-f55e9c313384.png)

### Do you see errors in the dev console? If yes, please include a screenshot

(To open the dev console in Google Chrome, press F12, or go to `View -> Developer -> Developer Tools`, and open the Console tab)

*Your answer*
NO Error
---
Its obvious am not that smart :) coding is getting me crazy here, please can someone guide me on the exact content of the config.js file. i will be most grateful, i keep getting error when i run the code you can see in the screenshot.

**Comment by vbaranov** [comment]
Try this config: https://github.com/poanetwork/ico-wizard/blob/08d77ffaf71f73d9f1665c5c7f8e28492220f92c/src/components/Common/config.js
Uncomment and please do not forget to change your crowdsale address.

**Comment by jacobabiola** [comment]
Thanks @vbaranov for taking the time, i added the configuration like you requested, but still getting error, please what am still missing, thank you very much for the support.
![screen shot 2018-01-11 at 1 04 01 pm](https://user-images.githubusercontent.com/24619534/34824814-48f5c474-f6d0-11e7-80a3-26a80ea1ec03.png)
![screen shot 2018-01-11 at 1 04 09 pm](https://user-images.githubusercontent.com/24619534/34824815-49390f5e-f6d0-11e7-942e-4485cc7dfd73.png)

**Comment by vbaranov** [comment]
Indentions in your 2nd screenshot are not the same, that in the demo at the link above. Just uncomment and change contract address to yourth.

**Comment by jacobabiola** [comment]
Ok thanks @vbaranov , let me try again.

**Comment by jacobabiola** [comment]
:) @vbaranov thanks a mill!, it worked
